.github/workflows/release.yml

```diff
@@ -0,0 +1,40 @@
+name: "Release a tag as draft"
+
+on:
+  push:
+    tags:
+      - w*
```

Comment:
This constraint passes when not only manual tagging but releasing from Github page.
I will make the change soon.